### PR TITLE
feat: store active table name

### PR DESCRIPTION
**Pull request by MH4GF**
Keep the active table in the global state, so that the TableDetail can also be opened from the left pane.

https://github.com/user-attachments/assets/dd0052aa-04bd-46b0-bd3d-e6a5ab1188b0

## TODO(will be separated other PRs)

- [ ] Change style TableNode when active
- [ ] Fit view when changed active table
- [ ] Move files of TableDetail

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx
Comment (MH4GF) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx:1]: 📝 Remove the Drawer part and just update the active tableName.

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.tsx [hunk 1]
```diff
@@ -29,9 +30,12 @@ export const ERDRenderer: FC = () => {
...
             </div>
-            <ReactFlowProvider>
-              <ERDContent nodes={nodes} edges={edges} />
-            </ReactFlowProvider>
+            <TableDetailDrawerRoot>
+              <ReactFlowProvider>
+                <ERDContent nodes={nodes} edges={edges} />
+              </ReactFlowProvider>
+              <TableDetailDrawer />
+            </TableDetailDrawerRoot>
```
Comment (MH4GF) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.tsx:38]: 📝 There is now only one TableDetail in the react render tree.

File: frontend/packages/erd-core/src/components/ERDRenderer/TableDetailDrawer/TableDetailDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import { DrawerContent, DrawerPortal, DrawerRoot } from '@liam-hq/ui'
+import {
+  type FC,
+  type PropsWithChildren,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react'
+// biome-ignore lint/nursery/useImportRestrictions: Fixed in the next PR.
+import { TableDetail } from '../ERDContent/TableNode/TableDetail'
```
Comment (MH4GF) [inline frontend/packages/erd-core/src/components/ERDRenderer/TableDetailDrawer/TableDetailDrawer.tsx:15]: As a large number of files will be changed, this will be done in a separate PR.

File: frontend/packages/erd-core/src/components/ERDRenderer/TableDetailDrawer/TableDetailDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import styles from './TableDetailDrawer.module.css'
+
+const ANIMATION_DURATION = 165
```
Comment (MH4GF) [inline frontend/packages/erd-core/src/components/ERDRenderer/TableDetailDrawer/TableDetailDrawer.tsx:18]: 📝 Defined here because we want to use duration on the React side as well.

**Review by junkisai — APPROVED** [review]
LGTM!!🎉
